packages/common/src/search/_internal/portalSearchItems.ts

```diff
@@ -347,6 +347,7 @@ export const WellKnownItemPredicates: IWellKnownItemPredicates = {
   $initiative: [
     {
       type: "Hub Initiative",
+      typekeywords: "hubInitiativeV2",
```

Comment:
Will this affect the current initiative?

Reply:
@vivzhang Jay and I discussed that in our 1:1 yesterday, and he didn't think it was an issue. With that said, that doesn't mean there isn't some case we missed. I could add a new entry, if we think that's better.

Reply:
I see, that's fine with me, especially since we are close to the release!